components/motor/gpio/motor_encoder.go

```diff
@@ -72,7 +73,7 @@ func WrapMotorWithEncoder(
 		return nil, err
 	}
 
-	single, isSingle := rutils.UnwrapProxy(e).(*encoder.SingleEncoder)
+	single, isSingle := rutils.UnwrapProxy(e).(*single.Encoder)
```

Comment:
[minor] I'd like `newEncodedMotor` to check that the motor type is incremental in this file now.